### ISSUE TITLE
Click Optgroup and select / deselect Options for Bootstrap 4.0

### ISSUE THREAD

**Issue by mshalomdave** [snapappointments/bootstrap-select#2290]
Hello I am trying to create a bootstrap multi-select using Bootstrap 4.0 where by I can select the optgroups and hence the options within that optgroups are selected. I have tried various ways including from Stack Overflow and they seem either to be outdated or not working. Could you help. Using Bootstrap-select version 1.13.9

**Comment by mshalomdave** [comment]
Here is my web page

> 

> <!DOCTYPE html>

<html>
<head>
  <title>Bootstrap-select Tests (Bootstrap 4)</title>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">

  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="../dist/css/bootstrap-select.css">

  <style>
    body {
      padding-top: 70px;
    }
  </style>
</head>
<body>
<select class="selectpicker" multiple>
    <optgroup label="Fruit">
      <option>Apple</option>
      <option>Orange</option>
    </optgroup>
    <optgroup label="Vegetable">
      <option>Corn</option>
      <option>Carrot</option>
    </optgroup>
  </select>


<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.bundle.min.js"></script>
<script src="../dist/js/bootstrap-select.js"></script>

<script>


</script>
</body>
</html>

> 

>

**Comment by caseyjhol** [comment]
Duplicate of #737. Please 👍 to vote and subscribe 🔉 for updates.
